src/tests/ftest/dfuse/caching_check.py

```diff
@@ -36,6 +36,7 @@ def test_dfuse_caching_check(self):
         :avocado: tags=hw,medium
         :avocado: tags=daosio,dfuse
         :avocado: tags=DfuseCachingCheck,test_dfuse_caching_check
+        :avocado: tags=test_caching_check
```

Comment:
This one already has the tag `test_dfuse_caching_check`

Reply:
Done.